lib/Ride.dart

```diff
@@ -241,27 +224,31 @@ class _TimeLineState extends State<TimeLine> {
       height: size,
       child: Icon(Icons.circle, size: 9.75, color: grey),
       decoration: BoxDecoration(
-          color: Colors.white, shape: BoxShape.circle, boxShadow: [dropShadow]),
+          color: Colors.white, shape: BoxShape.circle, boxShadow: [CarriageTheme.shadow]),
     );
   }
 
   Widget locationInfo(bool isPickup, DateTime time, String location) {
-    TextStyle directionStyle = TextStyle(
-        color: Color(0xFFA7A7A7), fontSize: 11, fontWeight: FontWeight.bold);
-
-    TextStyle locationStyle = TextStyle(
-        color: Colors.black, fontSize: 17, fontWeight: FontWeight.normal);
-
-    TextStyle timeStyle = locationStyle.copyWith(fontWeight: FontWeight.bold);
-
     return Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
-      Text(isPickup ? 'Pickup' : 'Dropoff', style: directionStyle),
+      Text(
+          isPickup ? 'Pickup' : 'Dropoff',
+          style: CarriageTheme.caption1.copyWith(color: CarriageTheme.gray3)
```

Comment:
Same thing with the theme. Does the gray3 correspond to something in the design (in which case my bad, please ignore) or is it just the 3rd gray color? As someone coming into the project I'm not sure what this theme represents.

Reply:
Discussed offline, didn't realize this related to the design system. likely okay then